tic_tac_toe/game.py

```diff
@@ -63,22 +71,40 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    for x in range(3):
```

Comment:
I think this is a pretty clever use of x, y coordinates. We just iterated through the rows and columns instead.